src/ontology/cl-edit.owl

```diff
@@ -4514,6 +4514,7 @@ SubClassOf(obo:CL_0000119 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002129
 
 # Class: obo:CL_0000120 (granule cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://medical-dictionary.thefreedictionary.com/granule+cell>) obo:IAO_0000115 obo:CL_0000120 "A neuron that is small in size. This general class includes small neurons in the granular layer of the cerebellar cortex, cerebral cortex nuerons that are not pyramidal cells and small axon-less neurons found in the olfactory bulb.")
```

Comment:
Could the genus reflect the classification (`CNS neuron (sensu Vertebrata)`) a bit more closely?

Maybe something like “A neuron of the vertebrate central nervous system that is...”?